pyiron_base/generic/hdfio.py

```diff
@@ -291,40 +291,50 @@ def copy_to(self, destination, file_name=None, maintain_name=True):
             FileHDFio: FileHDFio object pointing to a file which now contains the same content as file of the current
                        FileHDFio object.
         """
+        def _internal_copy(source, target, dest, h5_path):
+            if dest.h5_path[0] == "/":
```

Comment:
There are lots of places where "/" and summing strings for paths is directly done -- are these within a safe environment where they're already well wrapped, or is there danger on a Windows machine where we get "\" or "\\"? The tests are passing so I assume this if fine, I just want to do a final double-check

Reply:
This is only within the HDF5 file, so `/` is always save, still I agree we should extend `pathlib` to support accessing the HDF5 files and have a continuous interface. But that is most likely a larger project.